GVRf/Framework/src/org/gearvrf/GVRRenderData.java

```diff
@@ -321,22 +358,47 @@ public void setRenderingOrder(int renderingOrder) {
     }
 
     /**
-     * @return {@code true} if {@code GL_CULL_FACE} is enabled, {@code false} if
-     *         not.
+     * @return current face to be culled
+     *         	See {@link GVRCullFaceEnum}.
      */
-    public boolean getCullTest() {
-        return NativeRenderData.getCullTest(getNative());
+    public int getCullFace() {
+    	// Get cull face for base pass
+        return NativeRenderData.getCullFace(getNative(), 0);
+    }
+    
+    /**
+     * @param pass The rendering pass to query cull face state.
+     * @return current face to be culled
+     *         	See {@link GVRCullFaceEnum}.
+     */
+    public int getCullFace(int pass) {
+        return NativeRenderData.getCullFace(getNative(), pass);
     }
 
     /**
-     * Set the {@code GL_CULL_FACE} option
+     * Set the face to be culled 
      * 
-     * @param cullTest
-     *            {@code true} if {@code GL_CULL_FACE} should be enabled,
-     *            {@code false} if not.
+     * @param cullFace
+     *            {@code GVRCullFaceEnum.Back} Tells Graphics API to discard back faces,
+     *            {@code GVRCullFaceEnum.Front} Tells Graphics API to discard front faces,
+     *            {@code GVRCullFaceEnum.None} Tells Graphics API to not discard any face
+     */
+    public void setCullFace(int cullFace) {
+    	// Set cull face for base pass
+        NativeRenderData.setCullFace(getNative(), cullFace, 0);
+    }
+    
+    /**
+     * Set the face to be culled 
+     * 
+     * @param cullFace
+     *            {@code GVRCullFaceEnum.Back} Tells Graphics API to discard back faces,
+     *            {@code GVRCullFaceEnum.Front} Tells Graphics API to discard front faces,
+     *            {@code GVRCullFaceEnum.None} Tells Graphics API to not discard any face
+     * @param pass The rendering pass to set cull face state
      */
-    public void setCullTest(boolean cullTest) {
```

Comment:
it looks like you removed setCullTest() and replaced it with setCullFace().  While I like set setCullFace() better :), we should deprecate setCullTest()/getCullTest() first and then remove them in a later release.